panel/util/warnings.py

```diff
@@ -60,13 +60,20 @@ def deprecated(
     old: str,
     new: str | None = None,
     extra: str | None = None,
```

Comment:
```suggestion
    *
    extra: str | None = None,
```

So that we don't make a simple input error at some point.